project/scripts/bisect.scala

```diff
@@ -133,7 +133,7 @@ object ValidationScript:
   def tmpScalaCliScript(command: String, args: Seq[String]): File = tmpScript(s"""
     |#!/usr/bin/env bash
     |export JAVA_HOME=${sys.props("java.home")}
-    |scala-cli ${command} -S "$$1" --server=false ${args.mkString(" ")}
+    |scala-cli --cli-version=1.9.0 ${command} -S "$$1" --server=false ${args.mkString(" ")}
```

Comment:
hardcoding `--cli-version=1.9.0` here will always enforce the JVM launcher... which is slightly slower than its native counterparts.
That being said, it does ensure the script will work for everyone. 🤔

Reply:
A potential improvement would be to check for the version of `scala-cli` being used 

This command will print the launcher version, only
```bash
scala-cli version --cli-version
```

And then if it's <1.9.0, print a warning and default to the `--cli-version` override.